### DISCUSSION TITLE
How to position dropdown menu?

### DISCUSSION THREAD

**Discussion by MattHeslington** [huntabyte/shadcn-svelte#1098] [Help] [answered]
Hello,

Fantastic library. Thanks to all involved. 
I feel I've missed something obvious somewhere but is it possible to position the dropdown menu? At the moment it's centred at the bottom, but I'd really like it to be aligned to the right side of the trigger button. In melt-ui I would add 
```
positioning: {
	placement: 'bottom-end'
}
```
to the builder or whatever it's called. I've tried passing a different value to the 'sideOffset' prop and trying to position using css, without success.
Cheers,
Matt

**Comment by shyakadavis** [comment, marked as answer]
Hey, @MattHeslington 

I believe the `side` prop on `DropdownMenu.Content` is what you're looking for. 🙂 

https://www.bits-ui.com/docs/components/dropdown-menu#dropdownmenucontent

**Reply by MattHeslington** [reply]
Cheers, thanks for your help!

I knew I'd missed it somewhere. This is what I was looking for:

```
<DropdownMenu.Content side="bottom" align="end">
```

Thanks again!
